test/jdk/ProblemList.txt

```diff
@@ -528,12 +529,15 @@ javax/management/remote/mandatory/subjectDelegation/SubjectDelegation1Test.java
 
 # jdk_net
 
-java/net/MulticastSocket/NoLoopbackPackets.java                 7122846 macosx-all
-java/net/MulticastSocket/SetLoopbackMode.java                   7122846 macosx-all
+java/net/MulticastSocket/NoLoopbackPackets.java                 7122846,8308807 macosx-all,aix-ppc64
+java/net/MulticastSocket/SetLoopbackMode.java                   7122846,8308807 macosx-all,aix-ppc64
 
-java/net/MulticastSocket/Test.java                              7145658 macosx-all
+java/net/MulticastSocket/Test.java                              7145658,8308807 macosx-all,aix-ppc64
 
-java/net/ServerSocket/AcceptInheritHandle.java                  8211854 aix-ppc64
+java/net/ServerSocket/AcceptInheritHandle.java                  8211854,8308807 aix-ppc64
```

Comment:
Needed? Test was not mentioned in JDK-8308807.

Reply:
Indeed, this was a mistake - added a commit to revert: (0fe3eaecf3f39e2905a6f9b58e6736fc7de583e1)
Thomas